elements/pfe-sass/maps/_colors.scss

```diff
@@ -125,7 +125,7 @@ $pfe-colors: (
   surface--border--lightest:          $pfe-color--gray-100,
   surface--border--lighter:           $pfe-color--gray-100,
   surface--border:                    $pfelements--gray-light,
-  surface--border--darker:            $pfe-color--gray-300,
+  surface--border--darker:            $pfe-color--gray-600,
```

Comment:
this is updated in the colors PR: https://github.com/patternfly/patternfly-elements/pull/895/files#diff-dd8d4ca9242653965159fd5dcf649116R123